src/scss/components/_mobile-pop-over.scss

```diff
@@ -0,0 +1,78 @@
+/**
+ * Mobile pop overs
+ * --------------------------------
+ * Used for mobile filters pop over
+ *
+ * ================================= */
+
+$mobilePopOver-background       : $white;
+$mobilePopOver-padding          : spacing(x2);
+$mobilePopOver-border-color     : $grey--lightest;
+$mobilePopOver-shadow-color     : rgba($black, 0.12);
+
+.c-mobilePopOver {
+
+    @include media('<mid') {
+        left: 0;
+        top: 60px;
+        height: 0;
+        opacity: 0;
+        width: 100%;
+        position: fixed;
+        overflow: hidden;
+        z-index: zIndex(high);
+        background: $mobilePopOver-background;
+        transition: top 200ms ease, opacity 150ms ease;
+
+        &.active {
+            top: 0;
+            opacity: 1;
+            height: 100%;
+            overflow: visible;
+            padding: 72px $mobilePopOver-padding 96px $mobilePopOver-padding;
+        }
+    }
+}
+
+    .c-mobilePopOver-header,
+    .c-mobilePopOver-footer {
+        left: 0;
+        width: 100%;
+        position: absolute;
+        padding: $mobilePopOver-padding;
+
+        @include media('>=mid') {
+            display: none;
+        }
+    }
+
+    .c-mobilePopOver-header {
+        top: 0;
+        box-shadow: 0 3px 4px 0 $grey--lighter;
+    }
+
+    .c-mobilePopOver-header-actionFirst,
```

Comment:
Is this styling for the buttons?  Might be better to target with `nth-child` potentially to make it more future proof